content/release/whatsnew_2.0.0.html

```diff
@@ -208,7 +208,7 @@ <h2>
                 <li>Admin: Sort improvements for Specials and Featured pages.</li>
                 <li>Core: Addition of manufacturer's part number (MPN) to the products table.</li>
                 <li>Core: Addition of shipping tax rate to the orders table.</li>
-                <li>Core: CrowdIn support for language translations.  See <a href="https://github.com/zencart/zencart/discussions/6039">this Github discussion</a></li>
+                <li>Core: CrowdIn support for language translations.  See <a href="https://github.com/zencart/zencart/discussions/6039">this Github discussion</a> and <a href="https://docs.joomla.org/Crowdin">the CrowdIn docs</a></li>
```

Comment:
I'm not so sure that a link to Joomla is going to be helpful. Particularly in our own release docs.

Reply:
Will remove on next checkin.